tools/wheels/gfortran_utils.sh

```diff
@@ -107,47 +105,68 @@ function get_gf_lib_for_suf {
-    if [ ! -e "$out_fname" ]; then
-        curl -L "${GF_LIB_URL}/$fname" > $out_fname || (echo "Fetch of $out_fname failed"; exit 1)
-    fi
     [ -s $out_fname ] || (echo "$out_fname is empty"; exit 24)
     echo "$out_fname"
 }
 
 if [ "$(uname)" == "Darwin" ]; then
     mac_target=${MACOSX_DEPLOYMENT_TARGET:-$(get_macosx_target)}
     export MACOSX_DEPLOYMENT_TARGET=$mac_target
-    GFORTRAN_DMG="${GF_UTIL_DIR}/archives/gfortran-4.9.0-Mavericks.dmg"
-    export GFORTRAN_SHA="$(shasum $GFORTRAN_DMG)"
-
-    function install_arm64_cross_gfortran {
-        curl -L -O https://github.com/isuruf/gcc/releases/download/gcc-10-arm-20210728/gfortran-darwin-arm64.tar.gz
-        export GFORTRAN_SHA=4a1354e61294d5163609e83b6b2b082bd9a9bbdf
-        if [[ "$(shasum gfortran-darwin-arm64.tar.gz)" != "${GFORTRAN_SHA}  gfortran-darwin-arm64.tar.gz" ]]; then
-            echo "shasum mismatch for gfortran-darwin-arm64"
+    # Keep this for now as some builds might depend on this being
+    # available before install_gfortran is called
+    export GFORTRAN_SHA=13102b81ece217a30733fae0d7388c5085ff8d34
+
+    function download_and_unpack_gfortran {
+	local arch=$1
+	local type=$2
+        curl -L -O https://github.com/isuruf/gcc/releases/download/releases%2Fgcc-11.3.0/gfortran-darwin-${arch}-${type}.tar.gz
+	case ${arch}-${type} in
+	    arm64-native)
+	        export GFORTRAN_SHA=0cad7ec15ded04c8213dd697918389bebf156a73
+		;;
+	    arm64-cross)
+		export GFORTRAN_SHA=877cf498dd1fa796f5b2e0859c4581de9ed6b07b
+		;;
+	    x86_64-native)
+		export GFORTRAN_SHA=13102b81ece217a30733fae0d7388c5085ff8d34
+		;;
+	    x86_64-cross)
+		export GFORTRAN_SHA=93c14f4161653aaff8a163ca0bae8b46f32f8d6d
+		;;
+	esac
+        if [[ "$(shasum gfortran-darwin-${arch}-${type}.tar.gz)" != "${GFORTRAN_SHA}  gfortran-darwin-${arch}-${type}.tar.gz" ]]; then
+            echo "shasum mismatch for gfortran-darwin-${arch}-${type}"
             exit 1
         fi
         sudo mkdir -p /opt/
-        sudo cp "gfortran-darwin-arm64.tar.gz" /opt/gfortran-darwin-arm64.tar.gz
+        sudo cp "gfortran-darwin-${arch}-${type}.tar.gz" /opt/gfortran-darwin-${arch}-${type}.tar.gz
         pushd /opt
-            sudo tar -xvf gfortran-darwin-arm64.tar.gz
-            sudo rm gfortran-darwin-arm64.tar.gz
+            sudo tar -xvf gfortran-darwin-${arch}-${type}.tar.gz
+            sudo rm gfortran-darwin-${arch}-${type}.tar.gz
         popd
-        export FC_ARM64="$(find /opt/gfortran-darwin-arm64/bin -name "*-gfortran")"
-        export FC_LOC=/opt/gfortran-darwin-arm64/bin
-        local libgfortran="$(find /opt/gfortran-darwin-arm64/lib -name libgfortran.dylib)"
+	if [[ "${type}" == "native" ]]; then
+	    # Link these into /usr/local so that there's no need to add rpath or -L
+	    for f in libgfortran.dylib libgfortran.5.dylib libgcc_s.1.dylib libgcc_s.1.1.dylib libquadmath.dylib libquadmath.0.dylib; do
+                ln -sf /opt/gfortran-darwin-${arch}-${type}/lib/$f /usr/local/lib/$f
```

Comment:
Until there's a new openblas with this compiler built, you have to add

```suggestion
                ln -sf /opt/gfortran-darwin-${arch}-${type}/lib/$f /usr/local/lib/$f
                mkdir -p /usr/local/gfortran/lib
                ln -sf /opt/gfortran-darwin-${arch}-${type}/lib/$f /usr/local/gfortran/lib/$f
```

Reply:
THanks @isuruf for your help on this. I tried to set `DYLD_LIBRARY_PATH=/usr/local/lib`. If that doesn't work I'll do this.